### PR TITLE
rss_iperf_test: change the statistics's format

### PR DESCRIPTION
**Pull request by yanglei-rh**
Changed this script statistics's format due to the latest driver has not support the old format. And after discussion with developer, QE only need focus on the later version is enough, so remove the old test code.

ID:1993
Signed-off-by: Lei Yang leiyang@redhat.com

### CODE REVIEW

**Comment by yanglei-rh** [comment]
Test Pass:
 (1/2) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.rss_iperf_test.host2guest.q35: PASS (254.36 s)
 (2/2) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.rss_iperf_test.guest2host.q35: PASS (251.17 s)

**Comment by yanglei-rh** [comment]
Hello @PaulYuuu @heywji Could you please help me review this patch? Thanks in advance.

**Review by heywji — APPROVED** [review]
LGTM

1-Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.rss_iperf_test.host2guest.q35 Finshed 2024-02-21 14:52:16 2024-02-21 14:56:12 PASS 236.168955
2-Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.rss_iperf_test.guest2host.q35 Finshed 2024-02-21 14:56:12 2024-02-21 15:00:16 PASS 244.215371

**Comment by yanglei-rh** [comment]
Hi @leexiaohui32 Please help me review this patch, thanks a lot.

File: qemu/tests/iperf_test.py [hunk 1]
```diff
@@ -91,10 +91,8 @@ def rss_test():
         rss_test_cmd = utils_misc.set_winutils_letter(
             guest_session, params["rss_test_cmd"])
         output = guest_session.cmd_output(rss_test_cmd)
-        rss_statistics = output.splitlines()[1].split()
-        rxerrors = int(rss_statistics[-4])
-        rxmissed = int(rss_statistics[-2])
-        if not (rxerrors == 0 and rxmissed == 0):
+        rss_statistics = output.strip().split('\n')
+        if "Errors=0" and "Misses=0" not in rss_statistics:
```
Comment (PaulYuuu) [inline qemu/tests/iperf_test.py:95]: bool(“Errors=0”) always equal to `True`, I would suggest using `re` module.

Comment (yanglei-rh) [reply qemu/tests/iperf_test.py:95]: > bool(“Errors=0”) always equal to `True`, I would suggest using `re` module.

So you mean this line should be change to the following format, right?
`if re.search("rss_statistics", "Errors=0") and re.search("rss_statistics", "Misses=0"):`

Comment (PaulYuuu) [reply qemu/tests/iperf_test.py:95]: You are comparing strings, this is unreasonable. Please check how to use `re` module to match multiple pattern.

Comment (yanglei-rh) [reply qemu/tests/iperf_test.py:95]: > You are comparing strings, this is unreasonable. Please check how to use `re` module to match multiple pattern.

Ok, got it your point, I will update it ASAP.

Comment (yanglei-rh) [reply qemu/tests/iperf_test.py]: @PaulYuuu Please help me review it again, thanks a lot.

**Review by PaulYuuu — APPROVED** [review]

**Review by heywji — APPROVED** [review]
LGTM

```
1-Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35 Finshed 2024-02-27 10:41:05 2024-02-27 11:02:25 PASS 1280.25856
2-Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.rss_iperf_test.host2guest.q35 Finshed 2024-02-27 11:02:26 2024-02-27 11:07:24 PASS 298.94376
3-Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.rss_iperf_test.guest2host.q35 Finshed 2024-02-27 11:07:25 2024-02-27 11:11:53 PASS 267.858721
Summary: 
Finshed=3, PASS=3
```

**Pull request merged**